src/components/MainLayout.vue

```diff
@@ -33,7 +33,7 @@ export default {
         return {
           padding: '0px'
         }
-      }
+      } return {}
```

Comment:
Should be on a new line to fit better with general styling guidelines
```suggestion
      }
      return {}
```